server/services/referral.service.js

```diff
@@ -111,8 +111,8 @@ export const sendReferralInvite = async (invite) => {
   }
 };
 
-export const getReferralById = async (referralId) =>
-  Referral.aggregate([
+export const getReferralById = async (referralId) => {
+  const referral = await Referral.aggregate([
```

Comment:
Can we destructure this to make it easier to use:
```suggestion
  const [referral] = await Referral.aggregate([
```